src/css/custom.css

```diff
@@ -7,42 +7,79 @@
 @tailwind components;
 @tailwind utilities;
 
-/* You can override the default Infima variables here. */
-
 :root {
+  /* These variables mainly pulled from dotcom */
+  --white: #ffffff;
+  --black: #0b0d11;
+  --offblack: #121417;
+  --white-veil: #ffffff03;
+  --black-veil: #00000003;
+
+  --runtime: #70ffaf;
+  --runtime-medium: #00662c;
+  --runtime-dark: #172723;
+  --runtime-secondary: #ebff01;
+  --runtime-secondary-dark: #232711;
+
+  --deploy: #01c2ff;
+  --deploy-light: #7fe0ff;
+  --deploy-lighter: #fddcf2;
+  --deploy-lightest: #fde8f8;
+  --deploy-medium: #3d588f;
+  --deploy-dark: #0c212a;
+
+  --subhosting: #ff8a01;
+  --subhosting-dark: #251c11;
+
+  --gray-0: #9ea0a5;
+  --gray-1: #868789;
+  --gray-2: #56575a;
+  --gray-3: #25272b;
+  --gray-4: #191b1f;
+  --gray-5: #14161a;
+
+  --code-1: #01c2ff;
+  --code-2: #00a341;
+  --code-3: #ae01ff;
+  --code-4: #ea8e04;
+  --code-5: #ffd601;
+  --code-6: #01ff67;
+  --code-7: #db01ff;
+
+  /* Docusaurus theme overrides */
   --ifm-heading-font-weight: 600;
   --ifm-font-weight-bold: 600;
-  /*
-  --ifm-color-primary: #2e8555;
-  --ifm-color-primary-dark: #29784c;
-  --ifm-color-primary-darker: #277148;
-  --ifm-color-primary-darkest: #205d3b;
-  --ifm-color-primary-light: #33925d;
-  --ifm-color-primary-lighter: #359962;
-  --ifm-color-primary-lightest: #3cad6e;
-  --ifm-code-font-size: 95%;
-  --ifm-code-padding-horizontal: 0.5em;
-  --ifm-code-padding-vertical: 0.25em;
-  --ifm-navbar-search-input-background-color: #fff;
-  --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
-  --ifm-font-family-base: Inter, ui-sans-serif, system-ui, -apple-system,
```

Comment:
Did you mean to have the font family cut out? It's rendering the system font stack right now but I'm down to see what it looks like with what we chose on the marketing pages

Reply:
That was definitely a mistake. I was trying to clean up the code before pushing and must have gotten a little over-zealous. 🤦

Reply:
Oh wait—no, it's just manually set on the body element further up. Still, probably best to be consistent and use the variables they provide.